src/internal/document_api/mod.rs

```diff
@@ -768,12 +768,12 @@ impl TryFrom<&EncryptedDek> for EncryptedDekP {
             } => {
                 let mut proto_edek_data = EncryptedDekDataP::default();
 
-                proto_edek_data.set_encryptedBytes(encrypted_message.bytes()[..].into());
+                proto_edek_data.set_encryptedBytes(encrypted_message.bytes().to_vec().into());
```

Comment:
Was this due to a change in the `bytes` dependency? Can you explain what's going on here in some detail?

Reply:
They were already doing a `to_vec` internally, but they've now removed the From implementation for byte slices. So I just had to do the `to_vec` myself since they have a From for that.

Reply:
Understood. Thanks.